modules/mw.cx.MachineTranslationService.js

```
@@ -152,7 +161,98 @@ mw.cx.MachineTranslationService.prototype.fetchProvidersError = function () {
+				.attr('href', 'https://mdwiki.toolforge.org/Translation_Dashboard/auth.php?a=login')
+				.attr('target', '_blank')
+				.text('Translation Dashboard')
+		);
+		$('.cx-message-widget-details').text(" Then refresh the page");
+
+	}, 2000)
+
+}
+
+
+mw.cx.MachineTranslationService.prototype.fetchCXServerToken_issue = function () {
```

ESLint

[error] Identifier 'fetchCXServerToken_issue' is not in camel case.

(camelcase)